### ISSUE TITLE
Additions

### ISSUE THREAD

**Issue by rjmiller927** [drscotthawley/ml-audio-start#2]
Here are some content additions I have:

Tutorials:
- Neural Network Programming, Deep Learning with PyTorch (Learn how to program neural networks using PyTorch) https://deeplizard.com/learn/video/v5cngxo4mIg
- Machine Learning & Deep Learning Fundamentals (Good high level intro to ML concepts and how neural networks operate)
https://deeplizard.com/learn/video/gZmobeGL0Yg

Signal Processing Topics:
- Gradient Descent https://ml-cheatsheet.readthedocs.io/en/latest/gradient_descent.html , https://en.wikipedia.org/wiki/Gradient_descent , https://www.kdnuggets.com/2017/04/simple-understand-gradient-descent-algorithm.html

**Comment by drscotthawley** [comment]
Ryan, this is great, thanks!  
And now, just to get a little more GitHub experience in on this, how about submitting an edited version of README.md as a Pull Request, and I'll accept it and merge it in?

**Comment by rjmiller927** [comment]
Hi Scott,

I’m looking into doing a pull request but am having some difficulty in figuring it out. I’m probably missing something very basic, but when I click on New Pull Request it takes me to a page that says “Compare Changes” but I don’t see anything that actually allows me to edit that read me. I’ve got a screen shot of where I’m stuck…


[cid:image001.png@01D5B7ED.A7034840]



Also, I saw that Bulla approved us doing an independent study! I’ll register for that when registration re-opens.



Enjoy your holidays!



-Ryan

**Comment by drscotthawley** [comment]
For some reason it's not showing your screenshot.   Hmm. 

Ok, for now I'll go ahead and edit the file myself, but let's leave this Issue open so that when we get back from break, we can together review going through the PR process so you can do it in the future.

**Comment by drscotthawley** [comment]
Now that Ryan's an admin, we don't need to worry about this. :-)
